go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/valyala/quicktemplate
 
+go 1.11
+
 require (
-	github.com/klauspost/compress v1.4.1 // indirect
-	github.com/klauspost/cpuid v1.2.0 // indirect
 	github.com/valyala/bytebufferpool v1.0.0
-	github.com/valyala/fasthttp v1.2.0
+	github.com/valyala/fasthttp v1.12.0
```

Comment:
Maybe, it can be removed, because fasthttp is only used in the example, not by quicktemplate.